### DISCUSSION TITLE
Permissions on entry type level

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#9156] [Ideas]
> Created by: Ruben S (ruben@webs.nl) on 2016/04/28 09:44:43 +0000
> Votes at time of UserVoice import: 7

-----------------------------------

Restrict user groups to add/modify/delete certain entry types. Instead of the whole channel.

**Comment by davidhellmann** [comment]
@angrybrad 
Is it planned to bring this to CraftCMS 4 like the Field Rules?
Would be really great to say: User X have access to a Section and Specific Entry Types. 

Found nothing to this within the Roadmap or in Discord.

**Reply by brandonkelly** [reply]
@davidhellmann No current plans to address this, however in Craft 4 you can use the “Current User Condition” setting on field layout tabs and fields, to limit their access to only certain user groups.

**Reply by darwinsoftware** [reply]
+1 for this. We use a structure section on a multi-site install and I want to be able to restrict entry types on a per-site basis via permissions or site type. Not all sites should be able to create all entry types for a given entry in the structure.

**Comment by iuscare** [comment]
Is this planned for Craft 5, and if so, is there already a version lock for its development (e.g., 5.3/5.4)? Since the entryfication took place, this would make even more sense nowadays.

**Reply by alexanderbuergin** [reply]
I would love to see that option
